package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cmc-legal-rep-frontend",
-  "version": "1.0.0-alpha",
+  "version": "1.0.0",
```

Comment:
just a question, we were talking about version 2.0.0 shouldn't this be the same?